resources/assets/js/forms.js

```diff
@@ -111,6 +111,35 @@ ready(() => {
   });
 });
 
+/**
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled
+ */
+const DISABLE_ELEMENTS = [
+  'button', 'command', 'fieldset', 'input', 'keygen', 'optgroup', 'option', 'select', 'textarea'
+];
+ready(() => {
+  document.querySelectorAll('input[type="radio"]').forEach((radioElement) => {
```

Comment:
Can you please add some description about what this entire block does? It is really hard to follow…

Reply:
added more comments